app/models/github/task.rb

```diff
@@ -63,4 +57,15 @@ class Github::Task < ApplicationRecord
   before_create do
     self.uuid = SecureRandom.compact_uuid if self.uuid.blank?
   end
+
+  private
+  def update_track!
+    self.track_id = Track.where(repo_url: repo_url).pick(:id)
+  end
```

Comment:
The `update_xxx!` tends to signify an update will occur (e.g. the method uses `update(...)`). In this situation it doesn't actually update anything (because it's used in the validation).

So I think it shouldn't use "update" in the name, and probably not `!` either. Maybe: `set_track_id_from_repo`.

Reply:
Renamed